Bottom: 4c5821cdf3ef4483a569a1affb2118ccbbfcd140
Top:    95e5672b22649506129764ac6624328970ee6a54
Author: Lawrence D'Anna <lawrence_danna@apple.com>
Date:   2019-10-03 12:29:17 -0700

Refresh of allow-arbitrary-python-streams

---
diff --git a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
index 7cbca083abe..643e4612118 100644
--- a/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
+++ b/lldb/source/Plugins/ScriptInterpreter/Python/PythonDataObjects.cpp
@@ -1260,8 +1260,6 @@ public:
 
 namespace {
 class BinaryPythonFile : public PythonIOFile {
-  friend class PythonFile;
-
 protected:
   int m_descriptor;
 
@@ -1316,8 +1314,6 @@ public:
 
 namespace {
 class TextPythonFile : public PythonIOFile {
-  friend class PythonFile;
-
 protected:
   int m_descriptor;
